File: PHP/ProjectEconomy/src/entity/Conta.php
<?php
	class Conta{
		private $id;
		private $pessoa;
		private $usuario;
		private $saldo;
		
		public function get($var){
			return $this->$var;
		}
		public function set($var,$value){
			$this->$var = $value;
		}
	}
?>

File: JAVA EE/ProjectEconomy/src/entity/Conta.java
package entity;

public class Conta {
	private int id;
	private int pessoa;
	private int usuario;
	private double saldo;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public int getPessoa() {
		return pessoa;
	}
	public void setPessoa(int pessoa) {
		this.pessoa = pessoa;
	}
	public int getUsuario() {
		return usuario;
	}
	public void setUsuario(int usuario) {
		this.usuario = usuario;
	}
	public double getSaldo() {
		return saldo;
	}
	public void setSaldo(double saldo) {
		this.saldo = saldo;
	}
}
